src/gui/mzroll/eicwidget.cpp

```diff
@@ -1436,6 +1436,13 @@ void EicWidget::setCompound(Compound* c) {
 		setFocusLine(c->expectedRt);
 		selectGroupNearRt(c->expectedRt);
 	}
+	else {
+		//remove previous focusline
+		if (_focusLine && _focusLine->scene()) {
+			scene()->removeItem(_focusLine);
+			resetZoom();
```

Comment:
@shubhra-agrawal 
resetZoom() clears everything. attaching screenshots to explain what I mean

_without `resetZoom()`_
<img width="1280" alt="screen shot 2018-02-02 at 2 14 21 pm" src="https://user-images.githubusercontent.com/10898272/35724355-6e04bfb8-0823-11e8-80fe-6c27a27ab172.png">

_with `resetZoom()`_
<img width="1235" alt="screen shot 2018-02-02 at 2 17 25 pm" src="https://user-images.githubusercontent.com/10898272/35724476-dd2d5ef4-0823-11e8-8023-af6735f0c744.png">

Is this intentional ?

Reply:
@rish9511 I only tested this with MS data not MS2. Thanks for pointing this out!

Reply:
@rish9511 This is one corner case where it's failing i.e. only when we're dealing with MS2 data and there is an empty EIC and no expected Rt information. Seems to work for all other cases. I've checked with all possible combinations of MS1/MS2 data, EIC/no EIC and with/without expectedRt but feel free to test it out once.